website/layouts/partials/cells.html

```diff
@@ -0,0 +1,16 @@
+<ul class="c-cells">
+  {{ if (eq .Section "projects") }}
+  <!-- projects are ordered by a manifest on the repo to make it friendly for back end devs -->
+  {{ $projects := getJSON
+  "https://raw.githubusercontent.com/CodeYourFuture/immersive-go-course/main/projects/metadata.json"}}
```

Comment:
Is there no way to load this from the local filesystem or local serve, rather than needing to reach out to (an obsolete version) on github?

Reply:
There is, but I thought we wanted it to live here to be friendlier and more obvious to back end devs who are not concerned with the website?

Hugo can't see outside its folder, so to store it here we'd have to mv it into Hugo's data folder. We can do this, but I don't know what the benefit is particularly? Can you say more?

Reply:
A couple of things:
1. If you're developing locally, and edit this file, you won't see things actually update until your PR merges
2. If you're offline (e.g. on a plane), you lose the ability to use a dev version of the website

Maybe we can have `website/build.sh` make a hard-link (or if hugo can use it, a symlink), so that the file can live where it is but still be used by hugo? Or if not, I'd probably favour moving the metadata file to be checked in in the website directory :)

Reply:
I could mv it into website/data on build and I can query that instead?

Reply:
Sounds great, thanks!

Reply:
https://github.com/CodeYourFuture/immersive-go-course/pull/103/commits/207e30c9fc1044c7a62a34be63f3236473a760f2